aips/aip-standardize-private-keys.md

```diff
@@ -0,0 +1,229 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: (AIP title)
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.>
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: <Draft | Last Call | Accepted | Final | Rejected>
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: <Standard (Core, Networking, Interface, Application, Framework) | Informational | Process>
+created: <mm/dd/yyyy>
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Standardize Private Keys
+
+## Summary
+
+This AIP defines how private keys should be represented off-chain. It provides a way for private keys to be diffentiated
+from addresses, and defines a way to determine which key scheme is associated.
+
+### Goals
+
+> What are the goals and what is in scope? Any metrics?
+
+Private keys are indistinguishable from on-chain account addresses. This looks to change that by providing a solution to
+differentiate private keys from on-chain addresses.
+
+The main goal is to prevent people from accidentally leaking their private keys on-chain. This is commonly caused by
+sending
+funds to a private key. After that, someone may find this private key and rotate the account's key on-chain.
+
+The secondary goal is to keep this change to be as seamless as possible.
+
+> Discuss the business impact and business value this change would impact.
+
+This would reduce the number of accidentally lost keys on-chain.
+
+### Out of Scope
+
+> What are we committing to not doing and why are they scoped out?
+
+Encryption of private keys at rest, as that is orthogonal to private keys being confused for addresses.
+
+## Motivation
+
+> Describe the impetus for this change. What does it accomplish?
+
+This change should reduce the common case of users leaking their private keys on-chain.
+
+> What might occur if we do not accept this proposal?
+
+Users will continue at their normal rate of losing their private keys, and their accounts associated.
+
+## Impact
+
+> Which audiences are impacted by this change? What type of action does the audience need to take?
+
+This affects users, and the wallets that they use. Wallets are the primary store of private keys, and any application
+that is a wallet (mobile, extension, or in-app) or tool with a wallet (Aptos CLI) would need to adopt any new standards
+around private keys, with backwards compatibility.
+
+## Alternative solutions
```

Comment:
we really need to move alternative solutions to later after proposing the solution...